app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingContent.kt

```diff
@@ -100,8 +103,23 @@ fun VaultItemListingContent(
     LazyColumn(
         modifier = modifier,
     ) {
+        item {
```

Comment:
💭 Not something to block the PR, more just a thought about the order of the nesting, does this technically create an empty item every-time where if we were using positions(`listState.scrollTo(0)`) could throw of the the index?

Reply:
It does make an empty item. I was following the pattern on this screen but we probably should investigate this in the future.